### PR TITLE
Created timeline rows, dealt with duplication

### PR DESCRIPTION
**Pull request by rohan-bansal**
As title says!

### CODE REVIEW

File: client/src/components/WorkspacePanels/BoxStyle.css [hunk 1]
```diff
@@ -21,7 +21,7 @@
   display: flex;
   justify-content: center;
   align-content: center;
-  flex-direction: column;
+  flex-direction: row;
```
Comment (nayeonshin) [inline client/src/components/WorkspacePanels/BoxStyle.css]: Isn't `flex-direction` itself set to `row` by default? Or are we like overriding the value?

File: client/src/components/WorkspacePanels/MainWorkspacePanel/MainWorkspace.js [hunk 2]
```diff
@@ -1,189 +1,89 @@
...
+              music[3].map(({ id, file, name, type }) => (
+                <div className="track-object" key={id}>
+                  <div className="type-display">{type}</div>
+                  <div className="track-data">{name}</div>
+                </div>
+              ))}
+            {provided.placeholder}
+          </div>
+        )}
+      </Droppable>
```
Comment (nayeonshin) [inline client/src/components/WorkspacePanels/MainWorkspacePanel/MainWorkspace.js]: Do we need to hard-code this `Droppable` component multiple times according to the row num?

File: client/src/routes/WorkspaceRoute/Workspace.js [hunk 3]
```diff
@@ -1,30 +1,116 @@
...
+  loadedUniversalEffects,
+} from "../../components/WorkspacePanels/EffectsPanel/Effects";
+import {
+  Samples,
+  loadedProjectSamples,
+  loadedUniversalSamples,
+} from "../../components/WorkspacePanels/SamplesPanel/Samples";
+import MainWorkspace from "../../components/WorkspacePanels/MainWorkspacePanel/MainWorkspace";
+import { DragDropContext } from "react-beautiful-dnd";
+import React, { useState } from "react";
```
Comment (nayeonshin) [inline client/src/routes/WorkspaceRoute/Workspace.js]: Let's move this `import React` line to the beginning.

File: client/src/components/WorkspacePanels/MainWorkspacePanel/MainWorkspace.js (hunk 2 above)
Comment (rohan-bansal) [reply client/src/components/WorkspacePanels/MainWorkspacePanel/MainWorkspace.js]: good idea! i'll fix this

**Comment by nayeonshin** [comment]
Looks good! You're a coding genius.

**Pull request merged**